src/Microsoft.Identity.Abstractions/ApplicationOptions/ManagedIdentityOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Identity.Abstractions.ApplicationOptions
```

Comment:
Isn't the namespace of all the other classes/interfaces  Microsoft.Identity.Abstractions?

Reply:
changed